docs/Reference/CLI/Subcommands/Slashing-Protection.md

```diff
@@ -146,7 +146,186 @@ The file to export the slashing protection database to.
 
 Exports the database in the [validator client interchange format] format.
 
+## `repair`
+
+Repairs corrupted slashing-protection data files used by Teku.
+
+### check-only-enabled
+
+=== "Syntax"
+
+    ```bash
+    teku slashing-protection repair --checking-only-enabled[=<BOOLEAN>]
+    ```
+
+=== "Command Line"
+
+    ```bash
+    teku slashing-protection repair --checking-only-enabled=false
+    ```
```

Comment:
This will work if using all defaults, so the default library / database path is used - is that the intent?
Most of the time someone would have to specify either their normal config file or the paths in use to be able to find the files... basically the same behaviour as import/export slashing protection.

Reply:
I snagged these commands from [here](https://github.com/ConsenSys/teku/pull/4011#issuecomment-844554081). Should the `checking-only-enabled` command have a file specified in the example syntax?